packages/hooks/package.json

```diff
@@ -6,12 +6,16 @@
     "dependencies": {
         "@deriv/api": "^1.0.0",
         "@deriv/stores": "^1.0.0",
-        "react": "^17.0.2"
+        "react": "^17.0.2",
+        "lodash.groupby": "^4.6.0",
+        "lodash.pickby": "^4.6.0",
+        "moment": "^2.29.2"
```

Comment:
I think it's time for `@deriv/utils` 😅 We shouldn't have these dependencies here, Overtime this package will become the next `@deriv/shared` :(

We can move forward with this PR, I will check with @yashim-deriv and add a new package for `@deriv/utils` if possible and then you can update the code 🙇🏻